library/kong

```diff
@@ -25,8 +25,16 @@ GitFetch: refs/tags/3.4.2
 Directory: ubuntu
 Architectures: amd64, arm64v8
 
-Tags: 2.8.5-ubuntu, 2.8-ubuntu, 2.8.5, 2.8, 2
-GitCommit: 3fe94ba928bf75b385bd98aa2d5ad07b86fd3eb8
+Tags: 2.8.5-alpine, 2.8-alpine, 2.8.5, 2.8, 2
+GitCommit: cdf93ae2106f998a2245a3eee6814b1ae68781af
 GitFetch: refs/tags/2.8.5
 Directory: ubuntu
```

Comment:
Looks like this reverted back to the `ubuntu` directory instead of the `alpine` directory.

Reply:
Fixed, thank you.